Add OpenTelemetry Collector Builder docs (#2516)
open-telemetry/opentelemetry.io: content/en/docs/collector/distributions.md

weight: 25
---

An OpenTelemetry Collector distribution is a collection of components
(receivers, processors, connectors, exporters, extensions) assembled in a
binary.

The OpenTelemetry projects provides two official [pre-built distributions][] of
the Collector: core and contrib. While core contains a few selected components,
geared towards interoperability with other open-source projects, the contrib

> distribution includes verything under the sun, including vendor-specific

Author: Great catch, thank you!

> components. The components included in the [distributions][] can be found by in
> the `manifest.yaml` of each distribution.

Reviewer: ```suggestion
components. For the components included in a [distribution][distributions], see the distribution's `manifest.yaml` file.
```

While we expect the core distribution to be useful to a wide variety of
use-cases, we acknowledge that users might end up needing components that are
only available in the contrib distribution. Given that the attack surface and

> binary size of the contrib might be uncomfortable to most users, users of the

Reviewer: ```suggestion
binary size of the contrib distribution, it might be intimidating to most users. Users of the
```
